version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "7.1.0-SNAPSHOT"
+version in ThisBuild := "8.0.0-SNAPSHOT"
```

Comment:
I changed the version because that's what we had in the past as well for major Play releases (`6.0.0`, `7.0.0`, etc.)